lib/util.js

```diff
@@ -371,10 +371,13 @@ function formatValue(ctx, value, recurseTimes) {
   // Look up the keys of the object.
   var keys = Object.keys(value);
   var visibleKeys = arrayToHash(keys);
+  const symbolKeys = Object.getOwnPropertySymbols(value);
+  const enumSymbolKeys = symbolKeys
+      .filter((key) => value.propertyIsEnumerable(key));
```

Comment:
This would break if `value` does not inherit from `Object.prototype`. `Object.getOwnPropertyDescriptor(value, key).enumerable` should would work.